tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature

```diff
@@ -280,3 +280,192 @@ Feature: copying file using file id
       | dav-path                          |
       | /remote.php/dav/spaces/<<FILEID>> |
       | /dav/spaces/<<FILEID>>            |
+
+
+  Scenario Outline: sharee copies a file from project to personal space
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "project-space" with content "some data" to "textfile.txt"
+    And we save it into "FILEID"
+    And user "Alice" has sent the following space share invitation:
+      | space           | project-space |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | <space-role>  |
+    When user "Brian" copies a file "/textfile.txt" into "/" inside space "Personal" using file-id path "<dav-path>"
+    Then the HTTP status code should be "201"
+    And for user "Brian" folder "/" of the space "project-space" should contain these files:
+      | textfile.txt |
+    And for user "Brian" folder "/" of the space "Personal" should contain these files:
+      | textfile.txt |
+    Examples:
+      | space-role   | dav-path                          |
+      | Manager      | /remote.php/dav/spaces/<<FILEID>> |
+      | Space Editor | /remote.php/dav/spaces/<<FILEID>> |
+      | Space Viewer | /remote.php/dav/spaces/<<FILEID>> |
+      | Manager      | /dav/spaces/<<FILEID>>            |
+      | Space Editor | /dav/spaces/<<FILEID>>            |
+      | Space Viewer | /dav/spaces/<<FILEID>>            |
+
+
+  Scenario Outline: sharee copies a file between two project spaces
+    Given user "Brian" has been created with default attributes and without skeleton files
```

Comment:
what about similar scenario? maybe delete it or update it?
https://github.com/owncloud/ocis/blob/14e0a6a58828a47cf3d248b8996144ab0368dacc/tests/acceptance/features/apiSpacesDavOperation/copyByFileId.feature#L269

Reply:
Is it okay if I remove scenario: `Scenario Outline: copy a file between two project spaces` as copy (between two project spaces) feature will be tested in scenario: `Scenario Outline: sharee copies a file between two project spaces`? 
Though, scenario: `Scenario Outline: sharee copies a file between two project spaces` is for received project space it shouldn't matter I suppose.

Reply:
yes, you should delete the scenario and add this one.